src/window/clock/event_time_clock.rs

```diff
@@ -68,7 +68,7 @@ impl ClockBuilder<TdPyAny> for EventClockConfig {
 add_pymethods!(
     EventClockConfig,
     parent: ClockConfig,
-    py_args: (dt_getter, wait_for_system_duration),
+    signature: (dt_getter, wait_for_system_duration=Duration::zero()),
```

Comment:
`wait_for_system_duration` is not optional in our current api, so we shouldn't add a default value here:
`signature: (dt_getter, wait_for_system_duration)`